examples/language-modeling/main.py

```diff
@@ -168,7 +168,7 @@
         assert False, "please only specify one of format and deployment_device"
 
     if args.deployment_device is None and args.format is None:
-        args.format == "auto_round"
+        args.format = "auto_round"
```

Comment:
Could you please conduct more extensive testing on the code before merging it? Thank you!